lib/client/api.go

```diff
@@ -531,6 +531,10 @@ type ProfileStatus struct {
 	// AWSRoleARNs is a list of allowed AWS role ARNs user can assume.
 	AWSRolesARNs []string
 
+	// AllowedResourceIDs is a list of resources the user can access. An empty
+	// list means there are no resource-specific restrictions.
+	AllowedResourceIDs []types.ResourceID
```

Comment:
If there are some IDs in the list, does that mean my access within this profile is limited _only_ to resources with those IDs or can I also access any other resources that I could access before being granted the access request? Based on the comment, I'd assume the former.

If that's not the case, perhaps the comment should say that those are extra resources that I can access?

Reply:
It is the former, when you log in with a search-based request those are the only resources you can access.